x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_by_type_panel/columns.tsx

```diff
@@ -46,7 +48,20 @@ export const getAlertsTypeTableColumns = (): Array<EuiBasicTableColumn<AlertsTyp
       return (
         <EuiHealth color={ALERT_TYPE_COLOR[type as AlertType]}>
           <EuiText grow={false} size="xs">
-            {ALERT_TYPE_LABEL[type as AlertType]}
+            <CellActions
+              mode={CellActionsMode.HOVER}
+              visibleCellActions={4}
+              showActionTooltips
+              triggerId={CELL_ACTIONS_DEFAULT_TRIGGER}
+              field={{
+                name: 'event.type',
+                value: 'denied',
+                type: 'keyword',
+              }}
+              metadata={{ negateFilters: type === 'Detection' ? true : false }} // Detection: event.type != denied
```

Comment:
```suggestion
              metadata={{ negateFilters: type === 'Detection' }} // Detection: event.type != denied
```
Also, don't we have a constant for that `Detection` value?